src/main/java/hillel/spring/doctors/DoctorService.java

```diff
@@ -24,23 +26,30 @@
 import java.util.function.Predicate;
 
 @Service
+@Slf4j
 
 public class DoctorService {
     private final DoctorRepository doctorRepository;
     private final List<String> specializations;
 
     private final PetService petService;
+    private static String LOG = DoctorService.class.getCanonicalName();
 
     public DoctorService(@Value("${doctors.specializations}") String[] specializations,
                          DoctorRepository doctorRepository,
                          PetService petService) {
         this.specializations = List.of(specializations);
         this.doctorRepository = doctorRepository;
         this.petService = petService;
+
     }
 
 
-    public Doctor createDoctor(Doctor doctor) {
+    public Doctor createDoctor(Doctor doctor, Diplom diplom) {
+
+        log.debug(LOG, "creating doctor");
```

Comment:
My prediction is that log output won't contain the "creating doctor" phrase.
`LOG` variable is redundant here. Each log message contains class (or logger) name by default.